shared/src/types/api/room.dto.ts

```diff
@@ -23,3 +23,15 @@ export type PartnerDisconnectPayload = {
 export type PartnerLeavePayload = {
   userId: number;
 };
+
+export type SubmitPayload = {
+  code: string;
+  template: string;
+  language: Language;
+  inputs: string[];
+  output: string;
+};
```

Comment:
The frontend probably can only pass the questionId, language, and code over. Other than that we will have to retrieve it from the question service.